src/experiments/tests/prebid-magnite.ts

```diff
@@ -0,0 +1,28 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const prebidMagnite: ABTest = {
+	id: 'PrebidMagnite',
+	author: '@commercial-dev',
+	start: '2024-07-18',
+	expiry: '2024-09-30',
+	audience: 1 / 100,
+	audienceOffset: 0 / 100,
```

Comment:
Just need to double check if the audienceOffset needs to be changed? As far as I remember this is to not have a user in multiple variants at the same time or separate variant and control audience. Please if someone can add to this.

Reply:
As far as Im aware, thats right, in that it may be to stop certain users cross over into multiple different test groups.
Im not sure how this would be needed in this instance?

Reply:
Does that work the same way with server-side and client-side tests?

Reply:
With server-side tests, the offset is chosen for you, as our test groups have no overlap (ignoring the 50% group). With client-side tests, you select a portion of the audience using the audienceOffset and it is helpful to avoid clashing with other client-side tests that are running. Looking at our chart in tools for visualising this, it looks like there aren't any clashes, so leaving audienceOffset at `0 / 100` here is fine 👍

Reply:
That's a very useful explanation Dom! Every time I forget what audienceOffset do I will get back to this PR to check your comment 😄 Thank you.